hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/protocol/commands/ReplicateContainerCommand.java

```diff
@@ -41,6 +41,7 @@
 
   private final long containerID;
   private final List<DatanodeDetails> sourceDatanodes;
+  private int replicaIndex = 0;
```

Comment:
We discussed offline about alternative approach to get the replicaIndex. I am updating here for the discussion context.
We discussed that this may be ok and simpler, even though we are not using this at the DN side currently. We could potentially use this for validation purposes if needed. 

The alternative method is that we could add a methods at ContainerManager to get the replicaIndex and we can use that method to get replicaIndex for a given node. After discussion we felt this is ok to just add to command rather than going back to container manager.